### PR TITLE
Add further tests for existence of content

### PR DESCRIPTION
**Pull request by Jaifroid**
This is a simple PR to address #520.

### CODE REVIEW

File: www/js/app.js [hunk 1]
```diff
@@ -906,14 +906,15 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                     // The content is fully loaded by the browser : we can hide the spinner
                     $("#searchingArticles").hide();
                     // Deflect drag-and-drop of ZIM file on the iframe to Config
-                    var doc = iframeArticleContent.contentDocument.documentElement;
+                    var doc = iframeArticleContent.contentDocument;
+                    doc = doc ? doc.documentElement : null;
```
Comment (mossroy) [inline www/js/app.js]: It would be better to use different variable names here for the contentDocument and its documentElement to avoid confusion

**Comment by mossroy** [comment]
Maybe I spoke too soon : Firefox 45 UI tests seem to fail on this PR, even after restarting the build.
I don't see why for now...

**Comment by Jaifroid** [comment]
I’ve been getting that Firefox failure a lot, sometimes repeatedly, and then randomly it’s fixed. Nothing material has changed with this code, and I don’t see any typos. However, I’m about to push a commit that doesn’t re-use one variable, though it really shouldn’t make any difference...

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by Jaifroid** [comment]
OK, nothing to do with that variable, then (failed again). This is Travis, not this PR. I was getting precisely that failure: `Timed out while waiting for element <#articleList> to be visible for 20000 milliseconds.` for FF45 multiple times on previous PRs. It even failed once on `master` after we had merged a PR that had itself had already passed. You can still see that failure in the Commits for Master (see screenshot below), and you can visit the failed Travis log and you'll find the error is exactly the same as this one...

image

**Comment by mossroy** [comment]
All right.
In this case you can squash and merge this PR.
We should open a new issue for these failures on Travis.
After a few restarts, some builds manage to pass : it looks like a random issue

**Pull request merged**